web-server/v0.4/src/utils/parse.js

```diff
@@ -12,6 +14,12 @@ export const parseIterationData = results => {
         dataIndex: 'iteration_name',
         width: 150,
         key: 'iteration_name',
+        render: (text, record) => (
```

Comment:
We should render controller names next to each result name as opposed to the iteration name (we can assume all iterations from a result belong to the same controller)